.github/workflows/release-workflow.yaml

```diff
@@ -1,15 +1,9 @@
-name: release
+name: release-workflow
```

Comment:
separated into a separate workflow as we should only update the readme versions on a tag push